aiida_vasp/parsers/vasp.py

```diff
@@ -93,7 +93,8 @@ def get_dos_node(vrp, dcp):
         if not vrp or not dcp:
             return None
         dosnode = DataFactory('array')()
-        if vrp.pdos:
+        # vrp.pdos is a numpy array, and thus not directly bool-convertible
+        if len(vrp.pdos) > 0:  # pylint: disable=len-as-condition
```

Comment:
We might want to check for `vrp.pdos.size` instead.
This means it has to be guaranteed that `vrp.pdos` is always initialized as an empty `array`.
It can not allowed to be `None`.

Reply:
I agree, although of course ``len(None)`` also throws an error. But better to be explicit about it.